### PR TITLE
csound, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: media-sound/csound/csound-6.13.0.recipe [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	cmake .. \
+		-DCMAKE_BUILD_TYPE="Release" \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_BINDIR=$prefix/bin \
+		-DCMAKE_INSTALL_LIBDIR=$libDir \
+		-DCMAKE_INSTALL_LOCALEDIR=$dataDir/locale \
+		-DCMAKE_INSTALL_INCLUDEDIR=$includeDir \
+		-DCMAKE_INSTALL_MODULEDIR=$dataDir/cmake/Modules/Csound \
+		-DLUAJIT_INCLUDE_DIR="-I$(finddir B_SYSTEM_HEADERS_DIRECTORY)$secondaryArchSubDir" \
+		-DLUAJIT_LIBRARY="$(finddir B_SYSTEM_DEVELOP_DIRECTORY)/lib$secondaryArchSubDir/libluajit-5.1.a" \
```
[the remark this answers is not in the document]
Comment (korli) [reply media-sound/csound/csound-6.13.0.recipe]: Something like that:
```
-DLUAJIT_INCLUDE_DIR="$portPackageLinksDir/devel~libluajit_5.1$secondaryArchSuffix/$relativeIncludeDir/" \
-DLUAJIT_LIBRARY="$portPackageLinksDir/devel~libluajit_5.1$secondaryArchSuffix/$relativeDevelopLibDir/libluajit-5.1.a" \

```

File: media-sound/csound/csound-6.13.0.recipe [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	devel:libcsound64$secondaryArchSuffix = $libVersionCompat
+	devel:luaCsnd6$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	csound$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	boost169${secondaryArchSuffix}_devel
```
[the remark this answers is not in the document]
Comment (Begasus) [reply media-sound/csound/csound-6.13.0.recipe]: Not sure why it's in there (probably left-over from previous version?) Builds OK without ...

[the remark this answers is not in the document]
Comment (korli) [reply media-sound/csound/csound-6.13.0.recipe]: https://github.com/csound/csound/blob/master/dependencies/README#L29
devel boost package should be needed.

File: media-sound/csound/csound-6.13.0.recipe (hunk 1 above)
[the remark this answers is not in the document]
Comment (Begasus) [reply media-sound/csound/csound-6.13.0.recipe]: Actually luajit isn't used also, but somehow without it it doesn't find lua.h, need to investigate that

[the remark this answers is not in the document]
Comment (Begasus) [reply media-sound/csound/csound-6.13.0.recipe]: From one of the issues at csound:
> Closing this as I do not plain to maintain LuaJIT in Csound. If someone else wants it, feel free.

File: media-sound/csound/csound-6.13.0.recipe (hunk 2 above)
[the remark this answers is not in the document]
Comment (Begasus) [reply media-sound/csound/csound-6.13.0.recipe]: I don't see any dependencies for luajit also in that link @korli  :)

File: media-sound/csound/csound-6.13.0.recipe (hunk 1 above)
[the remark this answers is not in the document]
Comment (Begasus) [reply media-sound/csound/csound-6.13.0.recipe]: LUA_H_PATH not needed also ... https://github.com/csound/csound/commit/a97424795d82087b35a1852220579805594c0ce2

File: media-sound/csound/csound-6.13.0.recipe [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+	"
+BUILD_PREREQUIRES="
+	cmd:bison
+	cmd:cmake
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python
```
Comment (korli) [inline media-sound/csound/csound-6.13.0.recipe]: Should be cmd:python2.7

**Pull request merged**